packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx

```diff
@@ -166,11 +166,11 @@ export class PanelBase extends BaseComponent<IPanelProps, IPanelState> implement
               ignoreExternalFocusing={ignoreExternalFocusing}
               forceFocusInsideTrap={isHiddenOnDismiss && !isOpen ? false : forceFocusInsideTrap}
               firstFocusableSelector={firstFocusableSelector}
+              isClickableOutsideFocusTrap={true}
```

Comment:
What's the reasoning behind making always true?

Reply:
It's actually the same logic as before: both the new and old code make the prop default to true, not always be true.

Reply:
Since the focusZoneProps were moved below the isClickableOutsideFocusTrap prop, the focusZoneProps should still be able to override this isClickableOutsideFocusTrap={true} assignment.

Reply:
Oh I see, thank you for the clarification. Do either of you think there's potentially a better way to set this default? I'm just a little concerned that if Panel were refactored sometime in the future and the ordering of these props was changed, this line could be mistakenly placed below {...focusTrapZoneProps} and override the isClickableOutsideFocusTrap passed in.

Reply:
If not, no worries. This looks good to me otherwise.